internal/server/server.go

```diff
@@ -202,16 +186,14 @@ func registerUIRoutes(router *gin.Engine, opts UIOptions) {
 			req.URL.Scheme = remote.Scheme
 			req.URL.Host = remote.Host
 		}
+		proxy.ErrorLog = log.New(logging.NewFilteredHTTPLogger(), "", 0)
```

Comment:
@dnephin does this look right? Or should we pass in the `httpErrorLog` from `server.Run` ?

Reply:
This looks right to me